ambassador/ambassador/ir/ir.py

```diff
@@ -440,6 +443,69 @@ def add_mapping(self, aconf: Config, mapping: IRBaseMapping) -> Optional[IRBaseM
         else:
             return None
 
+    def cluster_ingresses_to_mappings(self, aconf):
+        cluster_ingresses = aconf.get_config("ClusterIngress")
+        if cluster_ingresses is None:
+            return
+
+        for ci_name, ci in cluster_ingresses.items():
+            self.logger.debug(f"Parsing ClusterIngress {ci_name}")
+
+            ci_rules = ci.get('rules', [])
+            for rule_count, ci_rule in enumerate(ci_rules):
+                ci_hosts = ci_rule.get('hosts', [])
+
+                mapping_host = ""
+                for ci_host in ci_hosts:
+                    if mapping_host == "":
+                        mapping_host = ci_host
+                    else:
+                        mapping_host += "|" + ci_host
+
+                ci_http = ci_rule.get('http', None)
+                if ci_http is not None:
+                    ci_paths = ci_http.get('paths', [])
+                    for path_count, ci_path in enumerate(ci_paths):
+                        ci_headers = ci_path.get('appendHeaders', [])
+
+                        ci_splits = ci_path.get('splits', [])
+                        for split_count, ci_split in enumerate(ci_splits):
+                            unique_suffix = f"{rule_count}{path_count}"
```

Comment:
I feel like this should be `f'{rule_count}-{path_count}'` to avoid collisions between, say, rule 1, path 21 and rule 12, path 1...

Reply:
makes sense, fixed